plugins/package-managers/cargo/src/main/kotlin/Cargo.kt

```diff
@@ -297,20 +297,33 @@ private fun parsePackageId(node: JsonNode) =
 
 private fun parseRepositoryUrl(node: JsonNode) = node["repository"].textValueOrEmpty()
 
+private val GIT_DEPENDENCY_REGEX = Regex("git\\+(https://.*)\\?(?:rev|tag|branch)=.+#([0-9a-zA-Z]+)")
+
 private fun parseSourceArtifact(
     node: JsonNode,
     hashes: Map<String, String>
 ): RemoteArtifact? {
-    if (node["source"].textValueOrEmpty() != "registry+https://github.com/rust-lang/crates.io-index") {
-        return null
+
```

Comment:
Nit: Please remove this leading empty line.

Reply:
Done